Dockerfile

```diff
@@ -1,12 +1,13 @@
 FROM python:3.10
 
+RUN pip install --upgrade pip
+
 COPY requirements.txt .
 
 RUN pip install -r requirements.txt
 
 COPY dataset/ dataset/
-COPY type-model/ type-model/
+# COPY models/ models/
```

Comment:
Users are instructed to put the models they might want to validate in this directory in the README. It also seems to be used in lines `171` and `187` of `train.py` for exactly this purpose.. Shouldn't this be uncommented?

Reply:
By Adding `-v ${PWD}/models:/models` to the docker run command as outlined in the README you should have the same effect but this allows the model to be also stored on the host system instead of only in the container. This means that if you train the model in the container you actually get the resulting model, instead that it is hidden in the container.